dotcom-rendering/src/web/components/Dropdown.tsx

```diff
@@ -171,6 +173,60 @@ const buttonExpanded = css`
 	}
 `;
 
+const badgeDiameter = 20;
+const notificationColor = '#e50014';
```

Comment:
Yeah sounds good. I didn't do this initially because even though the colours match the meanings don't (this isn't an error). But I think the fact that we're assigning it to a const with a name clears up any possible confusion.